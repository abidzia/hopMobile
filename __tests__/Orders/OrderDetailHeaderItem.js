import "react-native";
import React from "react";

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

import OrderDetailHeaderItem from "../../src/components/Orders/OrderDetailHeaderItem";

it("renders correctly", () => {
  const tree = renderer.create(<OrderDetailHeaderItem />);
  expect(tree).toMatchSnapshot();
});
