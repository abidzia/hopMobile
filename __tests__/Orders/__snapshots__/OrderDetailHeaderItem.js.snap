// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#ccc",
      "borderBottomWidth": 0.5,
      "borderTopColor": "#ccc",
      "borderTopWidth": 0.5,
      "flexDirection": "row",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 18,
          "marginVertical": 20,
        },
        Object {
          "flex": 1,
          "marginLeft": 25,
        },
      ]
    }
  >
    Item
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 18,
          "marginVertical": 20,
        },
        Object {
          "marginRight": 40,
          "textAlign": "right",
        },
      ]
    }
  >
    Quantity
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 18,
          "marginVertical": 20,
        },
        Object {
          "marginRight": 25,
          "textAlign": "right",
        },
      ]
    }
  >
    Amount
  </Text>
</View>
`;
